pkgs/tools/filesystems/wiimms-iso-tools/default.nix

```
@@ -0,0 +1,41 @@
+{stdenv, fetchurl, zlib, ncurses, fuse}:
+
+stdenv.mkDerivation rec {
+  name = "wiimms-iso-tools";
+  version = "3.02a";
+
+  src = fetchurl {
+    url = "https://download.wiimm.de/source/wiimms-iso-tools/wiimms-iso-tools.source-${version}.tar.bz2";
+    sha256 = "074cvcaqz23xyihslc6n64wwxwcnl6xp7l0750yb9pc0wrqxmj69";
+  };
+
+  buildInputs = [ zlib ncurses fuse ];
+
+  patches = [ ./fix-paths.diff ];
+  postPatch = ''
+    patchShebangs setup.sh
+    patchShebangs gen-template.sh
+    patchShebangs gen-text-file.sh
+  '';
+
+  NIX_CFLAGS_COMPILE = "-Wno-error=format-security";
+
+  configurePhase = ''
+    INSTALL_PATH="$out" ./setup.sh
+  '';
+
+  buildFlags = [ "INSTALL_PATH=$(out)" ];
```

```suggestion
  INSTALL_PATH = "$out";
```

variable in mkDerivation are automatically exported as variable in the shell

---

But `configurePhase` should be defined anyways as `./setup.sh`, isn't it?

---

I already tested it. Is not necessary. 🤩

I am amending the patch and force-pushing it.